.github/workflows/build_and_deploy_docs.yaml

```diff
@@ -0,0 +1,94 @@
+name: Build and deploy docs to Github Pages
+
+on:
+  push:
+    branches:
+      - dev
```

Comment:
I would keep this on push to `documentation` branch. Or at least `release`, but not dev.

Reply:
than make it 'release', no need for an extra branch... documentation should be always in sync anyway :)

Reply:
made it 'release' as discussed